app/services/attachments/create_service.rb

```diff
@@ -22,7 +22,51 @@ def execute
       authorize! @attachable.project, to: :update_sample? if @attachable.instance_of?(Sample)
 
       @attachments.each(&:save)
+
+      persisted_fastq_attachments = @attachments.select { |attachment| attachment.persisted? && attachment.fastq? }
+
+      identify_illumina_pe(persisted_fastq_attachments) if persisted_fastq_attachments.count > 1
+
       @attachments
     end
+
+    private
+
+    def identify_illumina_pe(attachments) # rubocop:disable Metrics/MethodLength,Metrics/CyclomaticComplexity,Metrics/AbcSize
```

Comment:
Can we update this method name so it's clear. The `pe` part of the method anyways.

Reply:
Addressed in b04242c